### DISCUSSION TITLE
ch_PP-OCRv4_det 原始超轻量模型的训练

### DISCUSSION THREAD

**Discussion by minushuang** [PaddlePaddle/PaddleOCR#12973] [Q&A]
用自己的数据集从头开始训练ch_PP-OCRv4_det原始超轻量模型，根据如下表格，设置配置文件后，开始训练

```python tools/train.py -c configs/det/ch_PPpOCRv4/ch_PP-OCRv4_det_cml.yml```

模型名称 | 模型简介 | 配置文件 | 推理模型大小 | 下载地址
-- | -- | -- | -- | --
ch_PP-OCRv4_det | 【最新】原始超轻量模型，支持中英文、多语种文本检测 | ch_PP-OCRv4_det_cml.yml | 4.70M | 推理模型 / 训练模型

训练保存的模型如下，不是超轻量模型，请问如何设置训练超轻量模型？

模型文件 | 大小
-- | --
latest.pdopt|55M
latest.pdparams|154M
latest.states|416k

**Comment by cuicheng01** [comment]
PP-OCRv4的骨干网络带了重参数化（rep）模块，你这边展示的这个是rep前的模型，动态图的pdparams会比较大，可以导出成静态图模型，就会比较小了，也就是真实使用的时候是比较小的。

**Comment by minushuang** [comment]
> PP-OCRv4的骨干网络带了重参数化（rep）模块，你这边展示的这个是rep前的模型，动态图的pdparams会比较大，可以导出成静态图模型，就会比较小了，也就是真实使用的时候是比较小的。

你好，请问如何将动态图转成静态图呢？
另外，我看readme里面，官方提供的推理模型时4.7M，训练模型是13M，我这训练模型100+M，通过动态转静态可以到13M吗？

**Comment by cuicheng01** [comment]
导出成静态图模型的化存储差不多下降4倍左右，你训练的字典有多大呢？

**Comment by minushuang** [comment]
@cuicheng01 训练的是det模型，det应该没有字典？

**Comment by minushuang** [comment]
@cuicheng01 导出静态图是tools/export_model.py这个脚本吗

**Comment by minushuang** [comment]
已导出，得到三个inference模型，分别是Teacher、Student和Student2，两个student模型确实是4.8M

**Comment by countryhotel** [comment]
你好，你在训练的时候使用 ch_PP-OCRv4_det_cml.yml

这个配置文件不会报错吗？

**Comment by minushuang** [comment]
按照文档设置好数据路径，一般不会出错

**Comment by tiandao011** [comment]
> 按照文档设置好数据路径，一般不会出错

我这边报 Backbone不能使用PPLCNetNew网络，请问你的paddleocr版本是多少？

**Comment by minushuang** [comment]
> > 按照文档设置好数据路径，一般不会出错
> 
> 我这边报 Backbone不能使用PPLCNetNew网络，请问你的paddleocr版本是多少？

将`PPLCNetNew`改成`PPLCNetV3`
